### PR TITLE
Fix distgen generation

### PR DESCRIPTION
**Pull request by zmiklank**
This commit should get the source code to the state before introduction of the new generator.

### CODE REVIEW

**Comment by zmiklank** [comment]
@fila43 Could you please check, if all supported versions are available in the sources now?

**Comment by zmiklank** [comment]
[test-all]

**Review by phracek — CHANGES_REQUESTED** [review]
Based on the https://src.fedoraproject.org/rpms/postgresql version 10 is out of scope for Fedora. In Fedora 35 we have version 13. Therefore Dockerfile.fedora for version `10` does not make sense.

Postgresql 10 will have EOL in December 2022 for RHEL7 and therefore I do not know if want to have it. But I am not against it.

File: 12/Dockerfile.centos7 [hunk 1]
```diff
@@ -0,0 +1,90 @@
+FROM quay.io/centos7/s2i-core-centos7
```
Comment (phracek) [inline 12/Dockerfile.centos7:1]:
```suggestion
FROM quay.io/centos7/s2i-core-centos7:latest
```

Comment (phracek) [reply 12/Dockerfile.centos7:1]: or `:centos7`. This could be better.

File: 13/Dockerfile.centos7 [hunk 2]
```diff
@@ -0,0 +1,90 @@
+FROM quay.io/centos7/s2i-core-centos7
```
Comment (phracek) [inline 13/Dockerfile.centos7:1]:
```suggestion
FROM quay.io/centos7/s2i-core-centos7:centos7
```

**Review by phracek — CHANGES_REQUESTED** [review]
Some nit picks.

**Pull request merged**